MGDAPI-5764 - removed all instances of local-rhoam-operator (#3257)
integr8ly/integreatly-operator: README.md

### Local and OLM installation type
If you installed RHOAM locally or via OLM then you can uninstall one of two ways:

> - for local installation use the namespace specified in the RHMI CR's spec.namespaceprefix value.

Reviewer: Not a big deal but this may be easier to read for people who aren't as familiar with kubernetes/openshift

```suggestion
- for local installation use the namespace that the RHMI CR was created in.
```
